### DISCUSSION TITLE
如何加密后仍能获取IDE的智能提示？

### DISCUSSION THREAD

**Discussion by richardwu0313** [dashingsoft/pyarmor#2015] [Q&A]
你好，
想问下在讲python 代码加密后，如何还能获取开发环境的智能提示呢？

**Comment by jondy** [comment]
不明白什么意思

**Reply by richardwu0313** [reply]
py代码加密后，文件里面都是如下代码，编辑器如pycharm无法提供语法提示。 比如。A.py下面有个B class， 没有加密前可以 from A import B, 加密后编辑器没办法提示import B，影响编码速度。

# Pyarmor 9.0.6 (trial), 000000, non-profits, 2024-12-19T10:58:44.974066
from .pyarmor_runtime_000000 import __pyarmor__
__pyarmor__(__name__, __file__, b'PY000000\x00\x03\x0b\x00\xa7\r\r\n\x80\x00\x01\x00\x08\x00\x00\x00\x04\x0
